src/BuildScriptGenerator/DotNetCore/DefaultAspNetCoreWebAppProjectFileProvider.cs

```diff
@@ -128,47 +133,9 @@ public string GetProjectFile(ISourceRepo sourceRepo)
         // To enable unit testing
         internal static bool IsAspNetCoreWebApplicationProject(XDocument projectFileDoc)
         {
-            if (IsAspNetCore30App(projectFileDoc))
-            {
-                return true;
-            }
-
-            var packageReferenceElements = projectFileDoc.XPathSelectElements(
-                DotnetCoreConstants.ProjectReferenceXPathExpression);
-            if (packageReferenceElements == null || !packageReferenceElements.Any())
-            {
-                return false;
-            }
-
-            var aspNetCorePackageReference = packageReferenceElements
-                .Where(packageRefElement =>
-                {
-                    if (!packageRefElement.HasAttributes)
-                    {
-                        return false;
-                    }
-
-                    var includeAttribute = packageRefElement.Attributes()
-                    .Where(attr => string.Equals(attr.Name.LocalName, "Include"))
-                    .FirstOrDefault();
-                    if (includeAttribute == null)
-                    {
-                        return false;
-                    }
-
-                    var value = includeAttribute.Value;
-                    return string.Equals(value, DotnetCoreConstants.AspNetCoreAppPackageReference)
-                    || string.Equals(value, DotnetCoreConstants.AspNetCoreAllPackageReference)
-                    || string.Equals(value, DotnetCoreConstants.AspNetCorePackageReference);
-                })
-                .FirstOrDefault();
-
-            if (aspNetCorePackageReference != null)
-            {
-                return true;
-            }
-
-            return false;
+            var webSdkProjectElement = projectFileDoc.XPathSelectElement(
```

Comment:
question: won't this change potentially make apps that run fine today break? If yes, we will need a plan to avoid breaking them

Reply:
Since all asp.net core versions since 1.0 use this attribute value "microsoft.net.sdk.web", we wouldn't be breaking existing apps.